Chapters/6_Qualified_identifiers.md

```diff
@@ -25,6 +25,15 @@ designates the function `generate_intput_stream` that is found at lines 9 to 15
 Notice that the notion of "line number" is not always meaningful: the content
 may be a binary file, or a file that uses non standard line termination character(s).
 
+### 6.1.2 Bytes qualifier
```

Comment:
While reading this, it occurred to me that we did not specify that the lines qualifier is 1-based.

We should take the chance of the addition of the bytes qualifier to explicitly state that:
- the lines qualifier is 1-based
- the bytes qualifier is 0-based

Reply:
Thanks, clarified in 5adfc538cb87732a162f39324e4eea02f51a3194